tests/unit/serve/runtimes/test_helper.py

```diff
@@ -270,6 +270,7 @@ def test_create_empty_doc_list_from_schema(transformation):
     class CustomDoc(BaseDoc):
         tensor: Optional[AnyTensor]
         url: ImageUrl
+        class_var: ClassVar[str] = "class_var_val"
```

Comment:
I think we need a better test for this. In this case only an empty DocList is created.

Reply:
I added classvars to more tests. Let me know if there are any more tests you think I should add it to.